### DISCUSSION TITLE
Investigate redundancy regarding criterion and Reserved-procurement authority tables

### DISCUSSION THREAD

**Discussion by AchillesDougalis** [eprocurementontology/eprocurementontology#574] [Alignment]
The ESPD uses the [Criterion ](https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/criterion)authority table to identify each criterion in the criterion Excel file one of which uses the code  "`shelt-worksh`". eforms uses the [Reserved-procurement](https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/reserved-procurement) authority table to indicate a similar concept `res-ws`. This possible redundancy needs to be looked into for the alignment of ESPD and eforms using the ePO.

**Comment by andreea-pasare** [comment]
For the time being, the reserved-procurement code list is not used in the ESPD-EDM model. However, we need a ParticipationCondition criterion (since this is not a SelectionCriterion), so we strictly follow ESPD and use the criterion codelist.
This will be reviewed in the alignment project.
![image](https://github.com/OP-TED/ePO/assets/88081600/e4c9f677-130b-4cd0-abc9-a656068b0255)
